### PR TITLE
Fix/pinned link style

### LINKED ISSUE #8046: [BUG] Pinned links look inactive
### Description

Pinned links on the links page look like they are inactive and when I was editing them at a glance I accidentally deleted one of them. If they are set to pinned they could still appear as being active I think.

EDIT: The approach should be to update the style of the pinned tag so that it is bright/active in appearance. Currently it appears dimmed so look inactive.

### Screenshots

buglinkfree

image

image (4)

This is a sketch of what it could look like (I'm sure it could be way better).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by akkshitgupta**
## Fixes Issue
Closes #8046 

## Changes proposed

UI of the pinned links in the profile is improved.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Screenshot 2023-08-31 at 12 22 31 AM

## Note to reviewers

Style of the pinned tag needed to be changed so that it doesn't look inactive anymore. Tag was having opacity 50% due to the parent container. So placed the tag outside the parent container and positioned it using the `absolute` property.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8387"></a>

### CODE REVIEW

**Comment by akkshitgupta** [comment]
Hey maintainers, can you please go through the pull request.

**Comment by amandamartin-dev** [comment]
> Hey maintainers, can you please go through the pull request.

Hello! Sorry for the delay. There are some merge conflicts in your PR. Can you resolve them so we can review.  If you have any trouble, let us know!

**Comment by amandamartin-dev** [comment]
@eddiejaoude was just checking this PR against the ticket and it's not clear to me if this UI approach was approved.  The last conversation resulting in keeping the original buttons and fixing the state of them as they appeared disabled due to the muted colors.  Please review when you have time

**Comment by eddiejaoude** [comment]
Thanks @akkshitgupta for the contribution and thanks @amandamartin-dev for reviewing 👍 

@akkshitgupta the design is not what agreed in the issue, please continue to use the current badge

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thanks @akkshitgupta for the contribution and thanks @amandamartin-dev for reviewing 👍 

@akkshitgupta the design is not what agreed in the issue, please continue to use the current badge

**Comment by akkshitgupta** [comment]
Hello @eddiejaoude sir, I made the changes as you requested. Can you please review the Pull Request. 

screenshot


**Note :** Style of the pinned tag needed to be changed so that it doesn't look inactive anymore. Tag was having opacity 50% due to the parent container. So placed the tag outside the parent container and positioned it using the absolute property.

**Comment by eddiejaoude** [comment]
Thank you @akkshitgupta for making the changes 👍 Instead the opacity and placing it outside then re-positioning, maybe we can use these red/green dots that tailing components use https://tailwindui.com/components/application-ui/lists/tables#component-2702e81c78671545b5184823a9fa1fc4

```js
const statuses = { Completed: 'text-green-400 bg-green-400/10', Error: 'text-rose-400 bg-rose-400/10' }
```

```js
<div className={classNames(statuses[item.status], 'flex-none rounded-full p-1')}>
   <div className="h-1.5 w-1.5 rounded-full bg-current" />
</div>
```

I can get the whole code from their example I that helps

**Comment by akkshitgupta** [comment]
Sure @eddiejaoude sir, we can go with this approach as well. However, I want to clear whether the tailing dot goes with `disabled` tag or with the entire link container

**Comment by eddiejaoude** [comment]
> we can go with this approach as well. However, I want to clear whether the tailing dot goes with `disabled` tag or with the entire link container

Good question 👍 I think if possible keep the badge also for now, but if the new red/green dot could go at the end of the container - if that is possible without too much code?

**Comment by akkshitgupta** [comment]
Screenshot

**Comment by eddiejaoude** [comment]
This looks great but it should not display on the profile page...

Screenshot 2023-08-30 at 23 27 24

File: components/user/UserLink.js [hunk 1]
```diff
@@ -84,6 +83,16 @@ export default function UserLink({
           {link.group}
         </span>
       )}
+      <div
```
Comment (eddiejaoude) [inline components/user/UserLink.js:86]: It needs `manage &&` before it as it should only display on the manage page

File: pages/account/manage/links.js [hunk 2]
```diff
@@ -105,7 +105,7 @@ export default function ManageLinks({ BASE_URL, username, links }) {
           additionalMessage="Your profile information has been saved successfully."
         />
 
-        <div className="flex gap-4">
+        <div className="flex gap-4 mb-5">
```
Comment (eddiejaoude) [inline pages/account/manage/links.js:108]: I think this change is not related to this issue?

Comment (akkshitgupta) [reply pages/account/manage/links.js]: This was to fix the consistency of the UI. But now removed.

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍  thank you!

**Comment by akkshitgupta** [comment]
Thank you @eddiejaoude sir for patiently coordinating through out this pull request.

**Pull request merged**